### PR TITLE
Rework and re-enable `webhookReply`

### PR DESCRIPTION
**Pull request by wojpawlik**
# Description

Typings of `sendMessage` indicate that it always returns sent `Message` on success, and our code relies on that to be able to delete it later (https://github.com/thedevs-network/the-guard-bot/issues/118#issuecomment-623101479).

After this change, only methods that always return `true` on success, providing no useful information, are eligible for `webhookReply`.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- `npm test` on Node.js v14.0.0

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: test/telegraf.js [hunk 1]
```diff
@@ -285,8 +284,8 @@ test.cb('should work with context extensions', (t) => {
 
 test.cb('should handle webhook response', (t) => {
   const bot = new Telegraf()
-  bot.on('message', async ({ reply }) => {
-    const result = await reply(':)')
+  bot.on('message', async (ctx) => {
+    const result = await ctx.replyWithChatAction('typing')
     t.deepEqual(result, { webhook: true })
```
Comment (wojpawlik) [inline test/telegraf.js:289]: This test seems to never fail!

Comment (wojpawlik) [reply test/telegraf.js:289]: ...because of `await` 😕

**Comment by KnorpelSenf** [comment]
Brief summary of how this could be solved:
- remove all current webhook reply configuration
- always use webhook reply as the first API request of the middleware, and use regular requests for follow-up API  calls, BUT
- only do this for methods where it makes sense (no meaningful return data or swallowed errors)
- let the user adjust what these methods are

**Comment by wojpawlik** [comment]
I'll leave those to a future pull request I think

> * remove all current webhook reply configuration
> * let the user adjust what these methods are

**Review by KnorpelSenf — APPROVED** [review]
Looks about right :+1:

**Pull request merged**